app/presenters/speedy_af/proxy/media_object.rb

```diff
@@ -140,6 +140,22 @@ def governing_policies
     @governing_policies ||= Array(attrs[:isGovernedBy]).collect { |id| SpeedyAF::Base.find(id) }
   end
 
+  def language
+    attrs[:language_code].map { |code| { code: code, text: LanguageTerm.find(code).text } } if attrs[:language_code].present?
+  end
+
+  def note
+    attrs[:note].map.with_index { |n, i| { note: n, type: attrs[:note_type][i] } } if attrs[:note].present?
+  end
+
+  def other_identifier
+    attrs[:other_identifier].map.with_index { |oi, i| { id: oi, source: attrs[:other_identifier_type][i] } if attrs[:other_identifier_type][i].present? }.compact if attrs[:other_identifier].present?
+  end
```

Comment:
This method is especially problematic because values outside of the media object are appended to this field in the solr_doc. If the order that things are added is maintained, it should be safe? May be better to split this field into what is included in the other_identifier accessor on the media object, and everything else. Unsure how much work splitting the field would be though.